lib/ruby_lsp/requests/support/common.rb

```diff
@@ -46,6 +46,23 @@ def visible?(node, range)
           range.cover?(loc.start_line - 1) && range.cover?(loc.end_line - 1)
         end
 
+        sig { params(node: Prism::Node, position: T::Hash[Symbol, T.untyped]).returns(T::Boolean) }
+        def node_covers?(node, position)
```

Comment:
Do we need to expose it now? If we only have signature help using it at the moment, I think it can be a private method there instead.